analytical_engine/core/parallel/parallel_property_message_manager.h

```diff
@@ -290,6 +291,22 @@ class ParallelPropertyMessageManager : public grape::MessageManagerBase {
                                                                   label, msg);
   }
 
+  /**
+   * @brief Get a bunch of messages, stored in a MessageInBuffer.
+   *
+   * @param buf Message buffer which holds a grape::OutArchive.
+   */
+  inline bool GetMessageInBuffer(grape::MessageInBuffer& buf) {
+    grape::OutArchive arc;
+    auto& que = recv_queues_[round_ % 2];
```

Comment:
It seems that this function get message in every other round？If so, please make the func name clear about this.

Reply:
How about `GetMessages`